### ISSUE TITLE
Treating cells as HTML elements

### ISSUE THREAD

**Issue by Mkranj** [cbailiss/basictabler#22]
Hi, I'm making a table where one column includes URL adresses. I'm displaying the table inside a Shiny app.
Can I parse cell contents of that column (or specific cells) as valid HTML?
E.g. 
`'<a href="https://www.google.com/">https://www.google.com/</a>'` should display as a link to Google.

Also, by using the `$getHtml()` method, I see that the cell contents look like this:
`&lt;a href="https://www.google.com/"&gt;https://www.google.com/&lt;/a&gt`

Basically the < and > symbols are being interpreted literally, perhaps that is preventing the conversion to actual HTML?
